src/Runner.Listener/BrokerMessageListener.cs

```diff
@@ -88,7 +90,8 @@ public async Task<CreateSessionResult> CreateSessionAsync(CancellationToken toke
                 try
                 {
                     Trace.Info("Connecting to the Broker Server...");
-                    await _brokerServer.ConnectAsync(new Uri(serverUrlV2), _creds);
+                    _credsV2 = _credMgr.LoadCredentials(allowAuthUrlV2: true);
```

Comment:
Is this bool used?

It looks like it's passed to `GetVssCredentials` but unused within that method
- https://github.com/actions/runner/blob/c1095ae2d100886a7865f466357f09ef30dec1b9/src/Runner.Listener/Configuration/CredentialProvider.cs#L36C85-L36C99

Reply:
it's only used in `OAuthCredential` 
https://github.com/actions/runner/blob/c1095ae2d100886a7865f466357f09ef30dec1b9/src/Runner.Listener/Configuration/OAuthCredential.cs